backend/api/manageUsersApi.ts

```diff
@@ -0,0 +1,267 @@
+import superagent from 'superagent'
+import * as querystring from 'querystring'
+
+import { Context } from '../interfaces/context'
```

Comment:
when referring to types on their own you could pull in the types on their own `import type { Blah }` or `import { type Blah }`